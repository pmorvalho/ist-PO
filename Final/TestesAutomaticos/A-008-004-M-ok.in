5
Mario Soares
ms@ps.pt
4
0
