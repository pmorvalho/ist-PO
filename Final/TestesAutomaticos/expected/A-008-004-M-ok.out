Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Nome do autor: Email do autor: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: Título: Objects in Space
Autor: Luke Skywalker/luke@jedi.com
Autor: Mario Soares/ms@ps.pt
Autor: Obi-Wan Kenobi/ob1@tatooine.geocities.com
Secções de topo: 3
Dimensão do documento (bytes): 77
Identificadores únicos: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Listar meta-informação
5 - Adicionar autor
6 - Listar secções
7 - Mostrar elemento de texto
8 - Editar
0 - Sair
Escolha uma opção: 